.github/workflows/label.yml

```diff
@@ -0,0 +1,71 @@
+name: Auto Label New Issues, PRs, and Discussions
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
```

Comment:
I think it would be better to run it on the `pull_request_target` event so that it also works on PRs from forks like this one.

More info on this is in https://github.com/actions/labeler/blob/506e1a00c6c05ba98ac585056a23f16073fa5915/README.md#permissions.

Reply:
Yes need to change that
I didn't test it with the PR from Fork I only test it with PR from repo only

Reply:
Yeah, I think that's good enough. The event payload should also be mostly identical between the two, so I don't expect any issues there.

Done in 431d16811acfae01b8078f52c147388855187648.